core/ui/public/i18n/en/translation.json

```
@@ -1718,5 +1731,27 @@
         "note_description": "A short note to help identify or describe this application.",
         "note_helper_text": "This note must be 100 characters or less",
         "filter_applications": "Filter applications"
+    },
+    "import_users": {
+        "import_data": "Import data",
+        "upload_csv_file_description": "Upload a CSV file containing user data. If the file includes group assignments, those groups will be imported as well.",
+        "csv_upload_label": "Upload file",
+        "csv_upload_helper_text": "Max 10000 rows. Supported format: CSV",
+        "manage_existing_users": "Manage existing users",
+        "manage_existing_users_tooltip": "Specify how to handle users that already exist in the cluster. You can either skip them or overwrite their data with the values from the file.",
+        "skip_existing_users": "Skip existing users",
+        "overwrite_existing_users": "Overwrite existing users",
+        "preview": "Preview",
+        "preview_description": "{users} users and {groups} groups will be imported. Review the data contained in the CSV file; only the first 5 rows are displayed.",
+        "too_many_users": "File exceeds size limit. Max 10000 rows are allowed",
+        "invalid_csv_format": "Invalid CSV format.",
```

No trailing dot at the end of validation errors
```suggestion
        "invalid_csv_format": "Invalid CSV format",
```